docs/cli.rst

```diff
@@ -0,0 +1,26 @@
+Command line interface
+======================
+
+In general, the only command that downstream repos will ever need to run is
+
+.. code-block:: bash
+
+    generate-bones all
```

Comment:
Is it important to note that the defaults won't put the scripts in the right place? Should the defaults be changed? Also related to what happens when `check-bones` fails due to version mismatch: it simply says to run `generate-bones`.

Reply:
You're right, should put something here about which directory to be in when calling `generate-bones`, and should update that message in `check-bones` now that the `all` option exists.

Reply:
Actually have you tested `all` on the other repos? I just tried it on `nengo-loihi` by cd'ing to `.ci` and running:

```
generate-bones --conf-file=../.nengobones.yml --template-dir="" all
```

It puts `.codecov.yml` and `.travis.yml` in the wrong place (inside `.ci` instead of one directory back). Also note that the empty string is needed in order to add the cwd to the templating paths. Ideally I think all that a user should need to do is run:

```
generate-bones all
```

from the root directory, and then it will use appropriate defaults for `--template-dir` and `--output-dir` depending on if it's `codecov-yml`, `travis-yml`, or the `ci-scripts`.

Reply:
Never mind the above ^ this was confusion on my part. See the below note.